fem/fespace.hpp

```diff
@@ -756,6 +756,9 @@ class QuadratureSpace
    /// Return the total number of quadrature points.
    int GetSize() const { return size; }
 
+   /// Return the order of the quadrature rules used by all elements.
```

Comment:
```suggestion
   /// Return the order of the quadrature rule used by all elements.
```

Shouldn't this be singular?

Reply:
Yes it should be thanks for catching that :)

Reply:
Plural was correct -- we have multiple rules for mixed meshes. Maybe "rule(s)" is more precise.

Reply:
I see. We can revert.

Reply:
Just make it "rule(s)"?

Reply:
Sure

Reply:
That's fine with me as well

Reply:
Done